src/classes/STG_PanelDataImportAdvancedMapping_CTRL.cls

```diff
@@ -344,4 +358,42 @@ public with sharing class STG_PanelDataImportAdvancedMapping_CTRL extends STG_Pa
     private void addPageMessage(ApexPages.Severity severity, String message) {
         ApexPages.addMessage(new ApexPages.Message(severity, message));
     }
+
+    /*******************************************************************************************************
+    * @description Action Method to enable Gift Entry
+    * @return null
+    */
+    public PageReference enableGiftEntry() {
+        Gift_Entry_Settings__c giftEntrySettings = UTIL_CustomSettingsFacade.getGiftEntrySettings();
+        if (!giftEntrySettings.Enable_Gift_Entry__c) {
+                giftEntrySettings.Enable_Gift_Entry__c = true;
+            try{
+                upsert giftEntrySettings;
+                addPageMessage(ApexPages.Severity.CONFIRM, System.Label.geDisableGiftEntryInfo);
+            }catch(DmlException e){
+                addPageMessage(ApexPages.Severity.ERROR,
+                        e.getDmlMessage(0));
+            }
+        }
+        return null;
+    }
+
+    /*******************************************************************************************************
+    * @description Action Method to disable Gift Entry
+    * @return null
+    */
+    public PageReference disableGiftEntry() {
+        Gift_Entry_Settings__c giftEntrySettings = UTIL_CustomSettingsFacade.getGiftEntrySettings();
+        if (giftEntrySettings.Enable_Gift_Entry__c) {
+                giftEntrySettings.Enable_Gift_Entry__c = false;
+            try{
+                upsert giftEntrySettings;
+                addPageMessage(ApexPages.Severity.INFO, System.Label.geEnableGiftEntrySuccess);
```

Comment:
I believe this renders the message "Gift Entry Enabled" when it should probably indicate that it's *disabled*